src/org/intellij/erlang/runconfig/ErlangRunConfigurationBase.java

```diff
@@ -41,8 +43,11 @@
 public abstract class ErlangRunConfigurationBase<RunningState extends ErlangRunningState> extends ModuleBasedConfiguration<ErlangModuleBasedConfiguration>
   implements RunConfigurationWithSuppressedDefaultRunAction {
   private ErlangDebugOptions myDebugOptions = new ErlangDebugOptions();
+  private String myWorkDirectory;
 
-  public ErlangRunConfigurationBase(String name, ErlangModuleBasedConfiguration configurationModule, ConfigurationFactory factory) {
+  public ErlangRunConfigurationBase(String name,
+                                    ErlangModuleBasedConfiguration configurationModule,
+                                    ConfigurationFactory factory) {
```

Comment:
Please avoid spurious formatting changes in your pull requests. It complicates browsing your changes.